.prettier.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  overrides: [
+    {
+      files: "**/*.sol",
+      options: {
+        printWidth: 120,
+        tabWidth: 2,
+      },
+    },
+    {
+      files: ["**/*.js", "**/*.ts", "**/*.vue"],
+      options: {
```

Comment:
There's a package called `@trivago/prettier-plugin-sort-imports` which extends prettier to sort imports, then I like configuring it like this:

```
// Standard prettier options
"printWidth": 100,
"tabWidth": 2,
// @trivago/prettier-plugin-sort-imports options
"importOrder": ["^react(.*)", "next/(.*)", "<THIRD_PARTY_MODULES>", "@/(.*)", "^[./]"],
"importOrderSortSpecifiers": true
```

This does two things:
1. Changes `import {b, a} from 'foof'` to be alphabetically sorted to `import {a, b} from 'foo'`
2. Sorts paths by the order specified in the array above. We'd just need to add the below snippet to tsconfig.json and then import from our own dirs using the `@/` syntax which I think is common. And `<THIRD_PARTY_MODULES>` is a specially recognized keyword by the package. 

Lmk what you guys think of this, but personally I find consistent import ordering nice, especially when it comes for ~no cost

```js
{
  "compilerOptions": {
    // --- snip ---
    "paths": {
      "@/*": ["./src/*"]
    }
  },
}

```

Reply:
I am down to use this

Reply:
(also noting that the two items I listed under "standard prettier options" are just an example, feel free to leave all the options you already have)

Reply:
i'm cool with this.

bikeshedding a bit but what about sorting package.json via sort-package-json?

Reply:
Isn't package.json sorted by default when doing `yarn add <pkg>`?

Reply:
sort-package-json sorts the scripts, packages, other fields

Reply:
ahh got it, up to you guys. those rarely change so I'm also ok with just sorting manually to avoid introducing another dependency (I like minimizing dependencies cause they've been compromised too often)

Reply:
I am fine either way, I think our other fields will be pretty small for the entire course of the project. So, manual management shouldn't be too burdensome, but sort-package-json could be good for the long term.

Reply:
Will leave this for you guys to decide/resolve, no objections to merging from me!